servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -689,6 +689,9 @@ void RenderForwardMobile::_pre_opaque_render(RenderDataRD *p_render_data) {
 	p_render_data->shadows.clear();
 	p_render_data->directional_shadows.clear();
 
+	const bool clear_directional_lights_by_region =
+			RD::get_singleton()->has_feature(RD::SUPPORTS_NATIVE_RENDER_AREA_CLEAR);
```

Comment:
```suggestion
	const bool clear_directional_lights_by_region = RD::get_singleton()->has_feature(RD::SUPPORTS_NATIVE_RENDER_AREA_CLEAR);
```

I like a short line as much as the next guy, but this is a little excessive